### ISSUE TITLE
$form doesn't work in Home

### ISSUE THREAD

**Issue by luckyshot** [luckyshot/CRMx#2]
```[Error] TypeError: 'undefined' is not a function (evaluating 'data.sort(createSorter(column))')
	sort (crmx.js, line 414)
	people (crmx.js, line 145)
	run (crmx.js, line 424)
	(función anónima) (pax, line 161)
	j (jquery.min.js, line 2)
	fireWith (jquery.min.js, line 2)
	ready (jquery.min.js, line 2)
	J (jquery.min.js, line 2)```

**Comment by webofnithin** [comment]
Hey did you fix it? i'm still getting this error?

**Comment by luckyshot** [comment]
Fixed in Version 0.3.0 fc90d5a32a2aef9b706e87b5f38197d138fe9020
